Helpers/get_results_from_criteria_and_data.py

```diff
@@ -0,0 +1,130 @@
+from datetime import date, timedelta
+from typing import List, Dict
+
+from Validators.checkin_checkout_validator import CheckinCheckoutValidator
+from graphql_objects import DatesCheck
+from mock_data import House
+
+
+class GetResultsFromSearchCriteriaAndData:
```

Comment:
`DatesCheckGetter`
use nouns for class names, verbs for function names

Reply:
Noted and done